OCPBUGS#9122: Improved configuring your firewall install info (#62848)
openshift/openshift-docs: modules/configuring-firewall.adoc

|`sso.redhat.com`
|443, 80
|The `https://console.redhat.com/openshift` site uses authentication from `sso.redhat.com`
[.small]
|===
+
[.small]
--

> 1. In a firewall environment, ensure that the `access.redhat.com` resource is on the allowlist. This resource hosts a signature store that `registry.access.redhat.com` uses for verifying subscriptions before you can pull images from the registry.

Reviewer: Something like:
```suggestion
1. In a firewall environment, ensure that the `access.redhat.com` resource is on the allowlist. This resource hosts a signature store required to verify images when pulling from `registry.access.redhat.com` .
```

Author: Thanks. Updated to:

> In a firewall environment, ensure that the `access.redhat.com` resource is on the allowlist. This resource hosts a signature store that a container tool requires for verifying images when pulling them from `registry.access.redhat.com`.
